### DISCUSSION TITLE
uhm..... what just happened?

### DISCUSSION THREAD

**Discussion by Vechea** [ppy/osu#15651] [Q&A]
![image](https://user-images.githubusercontent.com/77038686/141978537-dd7c1ba3-2ab2-4c2a-a21f-000f80f7a47a.png)

**Comment by bdach** [comment]
Whatever happened, it's the first time I've ever seen a table straight up missing from DB. And I don't think there is any migration that deletes it either.

Something has corrupted your database. At this point pretty much all that can be done is delete it fully and start anew.

Information regarding how you got into this state would be appreciated.

**Reply by Vechea** [reply]
Well that was partially my fault becuase I tried the migrate every file in lazer that I was playing and uh.... it was too taking too long (even by my own standards, like 30 minutes for like, 7 maps? with barely any attempt on them as well). I shut it down with alt+f4 and restarted it (which I think it was where the corruption thing started) and try to delete those beatmaps that I have in the list and continue the process. After a while it just pops ups like, "too many errors to show here, but click in this file to view more of it". 

I was migrating the client and data of it due to that, somehow the osu!lazer client was installed in the internal computer SSD that I'm running out of space for and tried to move it to external SSD and always getting hit with the error "Cant migrate this because app is working or using in %appdata%/osu/roaming and such"

**Reply by bdach** [reply]
Yes, interrupting the migration process unsafely will corrupt your data. So that's that.

On the other point:

> "Cant migrate this because app is working or using in %appdata%/osu/roaming and such"

Can't really say anything for sure given only a vague error message, but do you by any chance have any custom rulesets installed?
